code/modules/projectiles/projectile/special/meteor.dm

```diff
@@ -1,3 +1,120 @@
+#define DEFAULT_METEOR_LIFETIME 1800
+#define MAP_EDGE_PAD 5
+
+
+GLOBAL_VAR_INIT(meteor_wave_delay, 625) //minimum wait between waves in tenths of seconds
+//set to at least 100 unless you want evarr ruining every round
+
+//Meteors probability of spawning during a given wave
+GLOBAL_LIST_INIT(meteors_normal, list(/obj/projectile/meteor/dust=3, /obj/projectile/meteor/medium=8, /obj/projectile/meteor/big=3, \
+						  /obj/projectile/meteor/flaming=1, /obj/projectile/meteor/irradiated=3)) //for normal meteor event
+
+GLOBAL_LIST_INIT(meteors_threatening, list(/obj/projectile/meteor/medium=4, /obj/projectile/meteor/big=8, \
+						  /obj/projectile/meteor/flaming=3, /obj/projectile/meteor/irradiated=3)) //for threatening meteor event
+
+GLOBAL_LIST_INIT(meteors_catastrophic, list(/obj/projectile/meteor/medium=5, /obj/projectile/meteor/big=75, \
+						  /obj/projectile/meteor/flaming=10, /obj/projectile/meteor/irradiated=10, /obj/projectile/meteor/tunguska = 1)) //for catastrophic meteor event
+
+GLOBAL_LIST_INIT(meteorsB, list(/obj/projectile/meteor/meaty=5, /obj/projectile/meteor/meaty/xeno=1)) //for meaty ore event
+
+GLOBAL_LIST_INIT(meteorsC, list(/obj/projectile/meteor/dust)) //for space dust event
+
+GLOBAL_LIST_INIT(meteors_stress_test, list(/obj/projectile/meteor/tunguska))
```

Comment:
all of these don't need to be global lists

Reply:
that's how meteors already are, i guess I could make it one global list

Reply:
im saying they probably shouldnt be globs at all

Reply:
they are globals, they are accessed globally, they are globs

Reply:
like, i can't read your mind, if you want me to do something say exactly what you want done

Reply:
![image](https://user-images.githubusercontent.com/20987591/114624232-ad43a200-9cb0-11eb-9705-2b1976fd09c1.png)

Reply:
you haven't said what to replace them with

Reply:
meteor wave datums i guess? just anything that isn't stored as a glob when it's accessed in like 2 places.

Reply:
it's accessed in more than one place, therefore it goes in glob

Reply:
Put them in `SSaugury`.

Reply:
why augury?  augury is for spectating meteors, seems like an extremely weird place.

Reply:
You're right, they should be on their own subsystem or controller datum.